x-pack/legacy/plugins/integrations_manager/server/integrations.ts

```diff
@@ -175,16 +175,12 @@ function createIntegrationList(
   registryItems: RegistryList,
   integrationObjects: InstallationSavedObject[]
 ) {
-  const integrationList: IntegrationList = [];
-
-  for (const item of registryItems) {
-    const installedObject: InstallationSavedObject | undefined = integrationObjects.find(
-      ({ id }) => id === `${item.name}-${item.version}`
-    );
-
-    const integration = createInstallationObject(item, installedObject);
-    integrationList.push(integration);
-  }
+  const integrationList: IntegrationList = registryItems.map(item =>
```

Comment:
I'm hoping we don't need this `IntegrationList` type here, because `createInstallationObject` already returns this type by nature of all of the `registryItems` being the right type, right?

Reply:
Yeah. Seems fine without it https://github.com/elastic/kibana/commit/ac8c985d629c14f0f28683b86afdfaf05b05b15f